.github/workflows/ansible-test.yml

```diff
@@ -5,7 +5,12 @@
 # 3. For the comprehensive list of the inputs supported by the
 #    ansible-community/ansible-test-gh-action GitHub Action, see
 #    https://github.com/marketplace/actions/ansible-test.
-# 4. If you need help please ask in #ansible-community on the Libera.chat IRC
+# 4. If you want to prevent merging PRs that do not pass all tests,
+#    make sure to add the "check" job to your repository branch
+#    protection once this workflow is added.
+#    It is also possible to tweak which jobs are allowed to fail. See
+#    https://github.com/marketplace/actions/alls-green#gotchas for more detail.
+# 5. If you need help please ask in #ansible-community on the Libera.chat IRC
```

Comment:
I don't feel strongly about it but should this include Matrix?

Reply:
Dunno, maybe. But in a separate PR. I don't like mixing unrelated changes.

Reply:
I agree to both of you ;)